chalice/deploy/deployer.py

```diff
@@ -198,19 +198,20 @@ def deploy(self, config, chalice_stage_name=DEFAULT_STAGE_NAME):
         existing_resources = config.deployed_resources(chalice_stage_name)
         deployed_values = self._lambda_deploy.deploy(
             config, existing_resources, chalice_stage_name)
+        deployed_values.update({
```

Comment:
What was the reason for needing to insert these before the api gateway deploy as opposed to adding them at the end?